artifacts.go

```diff
@@ -32,14 +32,10 @@ import (
 // Version of the mender-artifact CLI tool
 var Version = "unknown"
 
+const LatestFormatVersion = 2
```

Comment:
Maybe adding some comment here would be nice. Something that we want to default to the latest supported version.